aten/src/ATen/native/DilatedMaxPool2d.cpp

```diff
@@ -69,59 +74,16 @@ bool ceil_mode) {
     set_output(1, {nbatch, nInputPlane, outputHeight, outputWidth}, {}, input.options().dtype(kLong), input.names());
   }
 }
-} // namespace meta
 
-namespace native {
-
-TORCH_IMPL_FUNC(max_pool2d_with_indices_out_cpu)
```

Comment:
For reference, refactoring preexisting functions like this makes review quite a bit harder for me, since I have to reverse engineer what the changes were made to the preexisting code as well as in the new code.

Reply:
verified this impl code was moved without change